### ISSUE TITLE
What do you think about Rust?

### ISSUE THREAD

**Issue by sindresorhus** [ChristopherBiscardi/ama#4]

**Comment by ChristopherBiscardi** [comment]
I'll start off by saying that I really enjoy programming in languages like Haskell, so my appreciation for Rust exists in the same plane. That said, I don't know what I'd use Rust for at the moment. Since it's a systems language, and most of the systems code around me right now is in Go (because Docker), I'd likely choose Go for any systems project. It doesn't seem like there's much of a focus on writing APIs, so my main use for Rust today would be to offload some computationally heavy operation from a node server.

The other use I could think of for Rust is if I needed to squeeze some performance out of a microservice which processed some data on-the-fly after fetching from Redis or Postgres.

So basically, I find Rust interesting and would choose it over C++ but need to find an appropriate project (suggestions accepted) on which to use it before further developing an opinion.
